geom/gdml/src/TGDMLWrite.cxx

```diff
@@ -1873,9 +1886,13 @@ XMLNodePointer_t TGDMLWrite::CreateBorderSurfaceN(TGeoBorderSurface * geoSurf)
    XMLNodePointer_t mainN = fGdmlE->NewChild(nullptr, nullptr, "bordersurface", nullptr);
    fGdmlE->NewAttr(mainN, nullptr, "name", geoSurf->GetName());
    fGdmlE->NewAttr(mainN, nullptr, "surfaceproperty", geoSurf->GetTitle());
-   // Cretate the logical volume reference node
+
+   // Cretate the logical volume reference nodes
    auto childN = fGdmlE->NewChild(nullptr, nullptr, "physvolref", nullptr);
    fGdmlE->NewAttr(childN, nullptr, "ref", geoSurf->GetNode1()->GetName());
+   fGdmlE->AddChild(mainN, childN);
+
+   childN = fGdmlE->NewChild(nullptr, nullptr, "physvolref", nullptr);
```

Comment:
Also unrelated to the treatment of dummy materials, can you please separate functionally the commits, otherwise it is going to be very difficult to track/debug the changes